### DISCUSSION TITLE
Button Sizes

### DISCUSSION THREAD

**Discussion by articillusion** [primefaces/community#2655] [Figma UI Kit]
Hello everyone, 

I was wondering if you could help me with finding where the button sizes sit under Figma UI Kit?

Below is the image to where it is shown in the guidelines however, I cannot see any reference in Figma to how to achieve this?

Looking forward to your help

Many Thanks
Fatih

<img width="1191" alt="Screenshot 2024-09-07 at 22 42 52" src="https://github.com/user-attachments/assets/968c1675-dd59-45e0-9c6a-bffa81ba99b0">

**Comment by onursenture** [comment]
Hi! Currently, there is no size option for buttons in the UI Kit. We will be adding it in future versions.

**Reply by andreeajircan** [reply]
Hi Onur,

Do you have an approximate timeline? Would it be in the next six months or so?

Thank you!

**Reply by onursenture** [reply]
Hi @andreeajircan, we'll be able to add it after PrimeVue 4.3.0, which is estimated to be released in December.
